src/elements/Image.tsx

```diff
@@ -18,6 +21,7 @@ export interface ImageProps extends CommonPathProps {
   href?: RNImageProps['source'] | string;
   preserveAspectRatio?: string;
   opacity?: NumberProp;
+  onLoad?: DirectEventHandler<ImageLoadEventData>;
```

Comment:
I think we want to type it with normal TS types, like here: https://github.com/software-mansion/react-native-screens/blob/5cfafbbf42a3b6ebcdb7772012f760c64bb305df/src/types.tsx#L189

Reply:
done.